xbmc/settings/Settings.h

```diff
@@ -121,6 +121,8 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static const std::string SETTING_VIDEOPLAYER_USEVAAPIMPEG2;
   static const std::string SETTING_VIDEOPLAYER_USEVAAPIMPEG4;
   static const std::string SETTING_VIDEOPLAYER_USEVAAPIVC1;
+  static const std::string SETTING_VIDEOPLAYER_USEVAAPIVP8;
```

Comment:
btw: I don't think we should continue touching general files for platform specific behaviour. I see zero benefit in having those constants. For querying the setting in a single place it requires two entries in two differetn files.